src/main/java/michal/kruczala/project/ptt/CompetitionNumberParser.java

```diff
@@ -1,172 +1,32 @@
 package michal.kruczala.project.ptt;
 
 import java.util.*;
+import java.util.stream.Stream;
 
 public class CompetitionNumberParser {
-    private int actualValue;
-    private String firstCompetitionNumber;
-    private static final String url = "turniej.php?nr=";
-    private List<String> competitionNumbers = ListInterface();
-    private int lastValue;
-    public List lookForCompetitionNumbers(String result) {
-
-        actualValue = result.indexOf(url);
-        lastValue = result.lastIndexOf(url);
-        firstCompetitionNumber = result.substring(actualValue);
-        competitionNumbers.add(firstCompetitionNumber);
-
-        AddingCompetitionNumbersToList(result, lastValue);
-        cleanDuplicatesInList();
-        return competitionNumbers;
+    String fraze = "nr=";
```

Comment:
Dlaczego w tej klasie są jakieś zmiany? Z tytułu i taska sądząc to nie powinno tu być żadnych zmian

Reply:
Zaszły zmiany, ponieważ na przestrzeni lat turnieje mają różne linki to też zaszła potrzeba rozszerzenia działania metody, a i zdarzają się numery turniejów trzycyfrowe więc trzeba tą metodkę było zrefaktoryzować :)